tests/test_newsheadlines.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import requests
+import os.path
+import sys
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+
+from app.newsheadlines import NewsHeadlines, NewsDisplay
+
+class TestNews(unittest.TestCase):
+    def setUp(self):
+        self.news = NewsHeadlines()
+        self.display = NewsDisplay()
+
+    def test_api_key_collection(self):
+        self.assertEqual(self.news.get_key(), '9b40cf73093e47e586cd61132f1d56fe')
```

Comment:
The main reason of putting the api key in the environment variables is to hide it from the users.
But you are exposing it here 🤔 !!!

Reply:
Ohh sorry, let me pick it from the env variables